README.md

```diff
@@ -123,8 +123,19 @@ on the same line (though some values can be broken over multiple lines).
 key = "value"
 ```
 
-Values must be of the following types: String, Integer, Float, Boolean,
-Datetime, Array, or Inline Table. Unspecified values are invalid.
+Values must be one of the following types.
```

Comment:
Grammar: likely this should be "Values must be of one of the following types." or "Values must have one of the following types."

Reply:
I'll go with "have one".